pytentiostat/routines.py

```diff
@@ -66,6 +89,19 @@ def startup_routine():
 
 
 def closing_routine(board, d9):
+    """
+    Called after experiment is finished. Function brings the potential back to 0 V and closes the board object.
+
+    Parameters
+    __________
+    board: board object for communication
+    d9: pin object for digital pin 9
+
+    Returns
+    _______
+    nothing
+
+    """
     # Prompt
     print("Experiment Complete! Closing...")
 
```

Comment:
``d9.write(0.5)`` should be changed to ``d9.write(RESTING_VOLTAGE)`` or sthg like that, and the global variable defined somewhere appropriate.